documentation/cli/docs/04_step_by_step.md

```diff
@@ -7,156 +7,189 @@ It is advised to do them all at least once in the given order to understand the
 ## Setup
 
 Initialise the wallet with a given node and a randomly generated mnemonic.
-```sh
-./wallet init --node [NODE API URL]
-> ...
+```sh title=Input
+./wallet init --node <NODE_API_URL>
+```
+```sh title=Output
 > INFO  Mnemonic stored successfully
 ```
 
 Create a main account.
-```sh
+```sh title=Input
 ./wallet new main
-> ...
+```
+```sh title=Output
 > INFO  Created account "main"
+```
+
+Exit from the "main" account.
+```sh title=Input
 > Account "main": exit
 ```
+Note: exiting from an account is omitted from now on for the rest of this guide.
 
 Create a savings account.
-```sh
+```sh title=Input
 ./wallet new savings
-> ...
+```
+```sh title=Output
 > INFO  Created account "savings"
-> Account "savings": exit
 ```
 
 ## Tokens
 
-Get some funds from the faucet to the main account.
-```sh
+Enter the "main" account, and get some funds from the faucet.
+```sh title=Input
 ./wallet main
-> Account "main": faucet [FAUCET ENQUEUE API URL]
-> ...
+
+> Account "main": faucet <FAUCET_ENQUEUE_API_URL>
 > Account "main": sync
-> ...
+```
+```sh title=Output
 > INFO  Synced: AccountBalance ...
-> Account "main": exit
 ```
 
 ### Send an amount
 
-Get an address from the savings account.
-```sh
+Enter the "savings" account, and get an address.
+```sh title=Input
 ./wallet savings
+
 > Account "savings": addresses
-> INFO  Address 0: [ADDRESS]
-> Account "savings": exit
+```
+```sh title=Output
+> INFO  Address 0: <tst123abc...>
```

Comment:
Default node is the testnet where we have rms, so I think we should use that instead (also in the places further below)